### ISSUE TITLE
HashMap user service should store user details in the cache?

### ISSUE THREAD

**Issue by m09526** [gchq/Palisade#160]
Currently, it just adds users to its local map and nothing is saved into the cache service.

**Comment by ac74475** [comment]
agreed this should also be storing user details into a cache service

**Comment by ac74475** [comment]
I don't think it is a bug as such, just another component that should be migrated to make use of the cache service to allow for persistence of data
